cmd/minishift/cmd/start_test.go

```diff
@@ -361,11 +362,11 @@ func TestDetermineIsoUrl(t *testing.T) {
 		in  string
 		out string
 	}{
-		{"", constants.DefaultB2dIsoUrl},
-		{"b2d", constants.DefaultB2dIsoUrl},
-		{"B2D", constants.DefaultB2dIsoUrl},
-		{"centos", constants.DefaultCentOsIsoUrl},
-		{"CentOs", constants.DefaultCentOsIsoUrl},
+		{"", minishiftUtil.GetLatestISOVersion("b2d")},
```

Comment:
can you motivate why this is not a good idea?
you mean you rather want a fixed version?